mnefun/_mnefun.py

```diff
@@ -1824,6 +1848,7 @@ def do_preprocessing_combined(p, subjects, run_indices):
                   '        %s' % bad_file)
             if not op.isdir(bad_dir):
                 os.mkdir(bad_dir)
+            #TODO 86 autobad after autoreject merge
```

Comment:
what does this mean?

Reply:
Thinking we could get rid of the automatic bad channel detection relying on arbitrary thresholds since autoreject module appears to also be able to "fix" bad channels.

Reply:
We should leave all old methods present/functional for backward compatibility